src/App.css

```diff
@@ -2,38 +2,27 @@
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.container {
+  display: flex;
+  justify-content: center;
+  margin-top: 5rem;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.card {
+  width: 20rem;
+  height: 15rem;
+  margin: 0%;
+  background: #928f8d;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.text {
+  text-align: right;
 }
```

Comment:
- [ ] OPTIONAL: Kindly make the text color similar ot the original. You should always stick to the given original design as if it is the client's requirement :pray: .

![image](https://user-images.githubusercontent.com/64894927/133310410-4b88098f-9972-4cfe-aa61-f3d4a1f1d171.png)

![image](https://user-images.githubusercontent.com/64894927/133310548-e2a55825-f950-44c7-b787-285d13f72e0d.png)

Reply:
Thanks made the suggested changes